CHANGELOG.md

```diff
@@ -72,6 +73,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
 - Add GeoTile and GeoHash Grid aggregations on GeoShapes. ([#5589](https://github.com/opensearch-project/OpenSearch/pull/5589))
 - Disallow multiple data paths for search nodes ([#6427](https://github.com/opensearch-project/OpenSearch/pull/6427))
+- [Segment Replication] Balanced primary shard distribution based allocation and rebalancing constraints ([#6422](https://github.com/opensearch-project/OpenSearch/pull/6422))
```

Comment:
Why are there 2 changelog entries here? With different text? Assuming this will be backported to 2.x then a changelog entry should only go in the `Unreleased 2.x` section.

Reply:
Yes, my bad. Added one more changelog entry after removing the `skip-changelog` label. Adding a single entry in unrelease `2.x` section.